packages/mural-account-chooser/src/components/account-chooser/index.tsx

```diff
@@ -246,19 +253,30 @@ const AccountChooser: React.FC<AccountChooserPropTypes> = (
               />
             )}
           </AccountChooserContent>
-          {hint && page === 'Sign in' && (
-            <UseDifferentEmail>
-              <NotYourEmail>
+          {hint && page === 'Sign in' ? (
```

Comment:
I think we miss a condition in storybook. We never see what the component looks like when the page is in 'SSO Option' mode.

I will try to add it. Because here I think it should be:
```
{page === 'Sign in' && (
  {hint ? <> : <>}
)}
```